### ISSUE TITLE
app is undefined for electron-updater

### ISSUE THREAD

**Issue by emilbruckner** [chentsulin/electron-react-boilerplate#1394]
I'm trying to use electron-updater from electron-builder, but get the following errors:

```
Uncaught TypeError: Cannot read property 'getVersion' of undefined
    at AppUpdater
…
```
&
```
Uncaught (in promise) TypeError: Cannot read property 'isReady' of undefined
    at AppUpdater.untilAppReady.resolve 
…
```

**The problem is:**
`this.app = require("electron").app` stays undefinded

**Comment by amilajack** [comment]
See https://github.com/chentsulin/electron-react-boilerplate/issues/1339#issuecomment-354141031

**Comment by emilbruckner** [comment]
I run everything in `app/main.dev.js` and still get the same error. @amilajack  Is `app/main.dev.js` what you mean by main process? Thanks

**Comment by emilbruckner** [comment]
Forget it, I'm just stupid

**Comment by emilbruckner** [comment]
I just never removed the error-creating code. 

What could be improved is the documentation. I had to understand the boilerplate before making updates work. (That actually doesn't sound like a bad thing.) 
It probably took me several hours to get updates to work properly. 
* I had to realize that electron-builder is used and how that works
* That I could use S3 instead of an update server
* Set S3 up
* Figure out that I have to run this in the main process and where that is
